docker_images/flair/app/pipelines/token_classification.py

```diff
@@ -27,21 +27,30 @@ def __call__(self, inputs: str) -> List[Dict[str, Any]]:
         """
         sentence: Sentence = Sentence(inputs)
 
-        # Also show scores for recognized NEs
-        self.tagger.predict(sentence, label_name="predicted")
+        self.tagger.predict(sentence)
 
         entities = []
-        for span in sentence.get_spans("predicted"):
-            if len(span.tokens) == 0:
-                continue
-            current_entity = {
-                "entity_group": span.tag,
-                "word": span.text,
-                "start": span.tokens[0].start_position,
-                "end": span.tokens[-1].end_position,
-                "score": span.score,
-            }
-
-            entities.append(current_entity)
+        for label in sentence.get_labels():
+            if isinstance(label, Label):
```

Comment:
From looking at https://github.com/flairNLP/flair/blob/master/flair/data.py#L371, we will always get `Label`s here. Do you have an example for which that's not the case? 

I tried using `flair/chunk-english-fast` but using `get_labels` outputs `Label` classes I think

```python
from flair.models import  Label, Sentence, SequenceTagger, Span

tagger = SequenceTagger.load("flair/chunk-english-fast")
sentence = Sentence("What's up in New York?")

for label in sentence.get_labels():
  print(label)
  print(isinstance(label, Label))
  print(isinstance(label, Span))
```

outputs

```
Span[0:1]: "What" → NP (1.0)
True
False
Span[1:2]: "'s" → VP (0.9997)
True
False
Span[2:3]: "up" → ADVP (0.3737)
True
False
Span[3:4]: "in" → PP (0.9987)
True
False
Span[4:6]: "New York" → NP (0.9996)
True
False
```

Reply:
Thanks, I will push a fix soon!